packages/testcontainers/src/container-runtime/clients/container/types.ts

```diff
@@ -3,3 +3,7 @@ export type Environment = { [key in string]: string };
 export type ExecOptions = { workingDir: string; user: string; env: Environment; log: boolean };
 
 export type ExecResult = { output: string; exitCode: number };
+
+export const ContainerStatus = ["created", "restarting", "running", "removing", "paused", "exited", "dead"] as const;
```

Comment:
I'd rename this to either `CONTAINER_STATUSES` or `containerStatuses`. Pascal case makes me think it's a type or a class

Reply:
Done 👍🏻 Liked `CONTAINER_STATUSES`, good point